### ISSUE TITLE
Loss function in training code

### ISSUE THREAD

**Issue by zhangconghhh** [peustr/WCA-Net#3]
In your paper, three loss are used for optimiztion, cross-entropy loss, wca loss and L2 regularization loss.
But I found the L2 regularization loss, is not used in train.py. 
https://github.com/peustr/WCA-Net/blob/97bf252b662909fc6564c800f0eba35827c56e80/train.py#L88
So, if the L2 regularization loss is used in the training process?

**Comment by peustr** [comment]
Hi there, the L2 regularization is the weight decay term in the optimizer:
https://github.com/peustr/WCA-Net/blob/main/train.py#L18
Thanks for looking at our work!
